src/ripple/core/Stoppable.h

```diff
@@ -186,9 +186,9 @@ class RootStoppable;
                                                              |
                                                          JobQueue
                                                              |
-        +--------+-----------+-----------+-----------+-------+---+----------+
-        |        |           |           |           |           |          |
-        |    NetworkOPs      |     InboundLedgers    |      OrderbookDB     |
+        +--------+-----------+-----------+-----------+-------+---+----------+----------+
+        |        |           |           |           |           |          |          |
+        |    NetworkOPs      |     InboundLedgers    |      OrderbookDB     |     GRPCServer
```

Comment:
`clang-format` is failing on my addition to this chart, not quite sure how to fix it.